egs/metrics/README.md

```diff
@@ -24,10 +24,10 @@ Until now, Amphion Evaluation has supported the following objective metrics:
   - Scale Invariant Signal to Distortion Ratio (SISDR)
   - Scale Invariant Signal to Noise Ratio (SISNR)
 - **Speaker Similarity**:
-  - Cosine similarity based on [Rawnet3](https://github.com/Jungjee/RawNet)
-  - Cosine similarity based on [Resemblyzer](https://github.com/resemble-ai/Resemblyzer)
-  - Cosine similarity based on [WavLM](https://github.com/microsoft/unilm/tree/master/wavlm)
-  - Cosine similarity based on [WeSpeaker](https://github.com/wenet-e2e/wespeaker) (👨‍💻 developing)
+  - Cosine similarity based on:
+    - [Rawnet3](https://github.com/Jungjee/RawNet)
+    - [Resemblyzer](https://github.com/resemble-ai/Resemblyzer)
+    - [WavLM](https://github.com/microsoft/unilm/tree/master/wavlm)
```

Comment:
Please modify the wavlm link to this one: https://huggingface.co/microsoft/wavlm-base-plus-sv

Reply:
fixed